hardware/Yeelight.cpp

```diff
@@ -117,7 +117,6 @@ void Yeelight::Do_Work()
 	_log.Log(LOG_STATUS, "YeeLight stopped");
 }
 
-
```

Comment:
Could you undo these formatting changes? It has nothing todo with Xiaomi, please make a separate pull request for this